client/components/videos-ui/index.jsx

```diff
@@ -20,6 +20,9 @@ const VideosUi = ( {
 	FooterBar,
 	areVideosTranslated = true,
 	intent = undefined,
+	headerTitle = false,
+	headerSubtitle = false,
+	headerOptionsList = false,
```

Comment:
Maybe the list should be an empty array? So that the type would always be an array, instead of a array|false.

Reply:
Done